usage/dist/pom.xml

```diff
@@ -98,9 +132,14 @@
                         <configuration>
                             <appendAssemblyId>true</appendAssemblyId>
                             <descriptors>
-                                <descriptor>src/main/config/build-distribution-archive.xml</descriptor>
+                                <descriptor>src/main/config/build-distribution.xml</descriptor>
                             </descriptors>
                             <finalName>brooklyn-${project.version}</finalName>
+                            <formats>
+                                <format>tar.gz</format>
+                                <format>zip</format>
+                            </formats>
+                            <attach>false</attach>
```

Comment:
I think we should stick to the default behaviour of attaching the artifacts here and remove the explicit `attach-artifact` goal instead, see comments in https://github.com/apache/incubator-brooklyn/pull/709.